packages/bbui/src/Modal/ModalContent.svelte

```diff
@@ -27,6 +27,7 @@
   export let secondaryButtonText = undefined
   export let secondaryAction = undefined
   export let secondaryButtonWarning = false
+  export let enableGrid = true
```

Comment:
After reading this, I think we'd be better to call it something more generic, yea. Maybe just call it `custom` - indicating it's a custom modal content rather than spectrum.